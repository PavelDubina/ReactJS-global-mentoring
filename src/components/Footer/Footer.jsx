import React from 'react'
import styles from './Footer.scss'

export const Footer = () => (
  <footer className={styles.container}>
    <span className={styles.title}>
      <b>netflix</b>
      roulette
    </span>
  </footer>
)
