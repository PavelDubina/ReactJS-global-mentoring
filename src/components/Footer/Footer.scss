.container {
  height: 56px;
  background-color: #424242;
  display: flex;
  align-items: center;
  justify-content: center;
}

.title {
  font-size: 20px;
  color: #f65261;
}
