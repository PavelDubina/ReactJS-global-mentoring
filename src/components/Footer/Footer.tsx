import React from 'react'
import styles from '../../../styles/Footer.module.scss'

export const Footer: React.FC = () => (
  <footer className={styles.container}>
    <span className={styles.title}>
      <b>netflix</b>
      roulette
    </span>
  </footer>
)
