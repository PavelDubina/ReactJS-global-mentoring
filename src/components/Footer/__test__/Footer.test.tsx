import React from 'react'
import { create } from 'react-test-renderer'
import { Footer } from '../Footer'

describe('Footer', () => {
  it('Should renders correctly', () => {
    const component = create(<Footer />).toJSON()
    expect(component).toMatchSnapshot()
  })
})
