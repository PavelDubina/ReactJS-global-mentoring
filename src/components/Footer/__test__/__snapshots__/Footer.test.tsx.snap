// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should renders correctly 1`] = `
<footer
  className="container"
>
  <span
    className="title"
  >
    <b>
      netflix
    </b>
    roulette
  </span>
</footer>
`;
